server/work-queue/queue.js

```diff
@@ -97,6 +97,10 @@ module.exports = {
   createVisit: function(o) {
```

Comment:
:heavy_check_mark: we want this to be fire-and-forget; the callback we care about connects the queue to the job